raiden/accounts.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser("~")
+    datadir = None
+    if sys.platform.startswith('linux'):
```

Comment:
Same reason I did not check for Windows there, since MVP is supposed to be only Linux and Mac. But I guess it would not hurt to have a more broad check. I believe in BSD the directory should be the same.